eclipse-theia/theia
# repo: update typescript references

Signed-off-by: Paul Maréchal <paul.marechal@ericsson.com>



#### What it does

1. Update typescript references.
2. Add a CI script to check if typescript references are in sync.

#### How to test

- Run `yarn test:references`: it should work.
- Change a reference in any package to something bogus (non-existant path).
- Run `yarn test:references`: it should fail.
